### DISCUSSION TITLE
ksmcc3 vs kusama?

### DISCUSSION THREAD

**Discussion by dzmitry-lahoda** [smol-dot/smoldot#586] [All]
I see in this folder chainspec  of statemine:

```
  "relay_chain": "ksmcc3",
```

while in cumulus repo

```
  "relay_chain": "kusama",
```

Same of other parachains.

I tried to use light-base to run unmodified parachain spec chain, it failed (chain not found).

Running full nodes with modified chain spec fails.

So how to reconcile difference. String replace?

**Comment by tomaka** [comment]
This is https://github.com/paritytech/cumulus/issues/1205

The TL;DR is that Cumulus has the word "kusama" hardcoded in its source code, while smoldot simply matches the `relay_chain` field of the parachain with the `id` field of the relay chain.
So yes, you can just put either `ksmcc3` or `kusama` everywhere.

Please keep in mind that there's no need to centralize chain specifications. Copy-pasting chain specs around is a perfectly correct way of doing it.
